samples/opencl/CMakeLists.txt

```diff
@@ -0,0 +1,66 @@
+include(FindOpenCL.cmake)
```

Comment:
You should not require CMake version here, only check and enable/disable these samples in case of version 3.1+.

```
if(CMAKE_VERSION VERSION_LESS "3.1")
  message(STATUS "OpenCL samples require CMakes 3.1+")
  return()
endif()
```

P.S. Yes, this sample will not be checked on public OpenCV buildbot.